### PR TITLE
Capstone project - Guitar Jamming

### PR DESCRIPTION
**Pull request by tarek102**
This PR is about the Capstone project.
The project is about Guitar jam gathering.
Created index.html and about.html
guitar players (speakers) are created dynamically through JavaScript in dynamic.js
The rest of the JavaScript is in script.js
update README.md

Loom video: https://www.loom.com/share/03d0018a4cf84964b3ee0bf9e2da1b83

New Loom video: https://www.loom.com/share/267dbfda6bc845a9aeea25bd0fb262e9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+
+- This template was designed by Cindy Shin
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+## Live Demo
+
+[Live Demo Link](https://tarek102.github.iocapstone-project-Jamming-page/)
```
Comment (Davidosky007) [inline README.md]: * Kindly fix the live demo link as its broken, below ⬇️ is a screenshot 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @tarek102 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights ⭐ 
* No linter errors ✔️ 
* Well written PR description ✔️ 
* Html best practices used 💯 

## Required Changes ♻️
* Nice Job with the video presentation, but The video lacks a highlight about a piece of the code that you find interesting or you are proud of making.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (tarek102) [reply README.md]: thank you @Davidosky007 for the review, I just wanted to clarify before I submit a new PR, should I record a new video explaining everything plus the piece of the code that I found interesting? or should I just update the live demo with the correct one?

**Review by AshakaE — APPROVED** [review]
Hi @tarek102 , 

Thanks for making the requested change 👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**